skunkworks_crow/src/main/java/org/odk/share/views/ui/hotspot/HpSenderActivity.java

```diff
@@ -447,11 +448,25 @@ private void createAlertDialog(String title, String message) {
     private void checkLocationPermission() {
 
         if (Build.VERSION.SDK_INT < Build.VERSION_CODES.M ||
-                checkSelfPermission(Manifest.permission.ACCESS_FINE_LOCATION) == PackageManager.PERMISSION_GRANTED) {
+                checkSelfPermission(Manifest.permission.ACCESS_COARSE_LOCATION) == PackageManager.PERMISSION_GRANTED) {
```

Comment:
Why changing the fine location to coarse location, and have you tested transferring forms ater this change?

Reply:
I understand what you are saying, but why we are using ACCESS_FINE_LOCATION in condition check but using ACCESS_COARSE_LOCATION while requesting permission.

Reply:
We're using Course Location at both the places, can you point me to the actual line where you have seen this issue?

Reply:
I am saying why are we using FINE_LOCATION in the review you mentioned above, but COARSE_LOCATION at other places.

Reply:
There could be a miss while reviewing the code, as much as I remember when you ask for location permission it shows only one dialog and that allows both the permissions but if in our case if we need only one of the permission to execute the method or the piece of code that I think we should stick with the same permission not both of them. Thanks @naman653  for poinitng this out. It would be helpful if you can open a issue regarding this.

Reply:
Happy to do that. But for which permission we should go for?

Reply:
That needs to be checked, if we're using both the permissions in our code then it's in our best interest to check for both of them while asking to the user.

Reply:
Sorry for late replying, as I was busy with my mid terms. I think we should go for Coarse Location as it was being used before, and was working just fine before.

Reply:
Also, tell me what should be done with this PR.

Reply:
@lakshyagupta21 Please review